packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -63,5 +63,16 @@ export const FUTURE_FLAGS = {
   [STACK_RELATIVE_EXPORTS_CONTEXT]: 'true',
 
   // We will advertise this flag when the feature is complete
-  // [NEW_STYLE_STACK_SYNTHESIS]: 'true',
+  // [NEW_STYLE_STACK_SYNTHESIS_CONTEXT]: 'true',
 };
+
+const FUTURE_FLAGS_DEFAULTS: { [key: string]: boolean } = {
```

Comment:
Add a comment explaining what this map is. It's a bit confusing that we have two maps with opposite values (one all true and the other all false). I am wondering if we should combine those maps to be slightly more structured?

```ts
export const FEATURE_FLAGS = {
  [ENABLE_DIFF_NO_FAIL_CONTEXT]: {
    newProjects: true,
    defaultValue: false
  }
};
```

I am also over with just adding a comment ;)

WDTY?

Reply:
I initially went down this path. The combination that `FUTURE_FLAGS` is exported and that one of the flags not yet part of the new project template makes the code more complicated than it needs to be.

What I have now is non optimal but feels pragmatic.